src/test/java/org/folio/rest/impl/PasswordResourceTest.java

```diff
@@ -13,102 +14,197 @@
+import static org.folio.services.validator.util.ValidatorHelper.RESPONSE_VALIDATION_RESULT_KEY;
+import static org.folio.services.validator.util.ValidatorHelper.VALIDATION_INVALID_RESULT;
+import static org.folio.services.validator.util.ValidatorHelper.VALIDATION_VALID_RESULT;
+
 @RunWith(VertxUnitRunner.class)
 public class PasswordResourceTest {
-  private static final String POST_VALIDATION_CREDENTIALS = "{\"password\":\"12345\"};";
-  private static final String VALIDATION_PATH_QUERYABLE = "/password/validators?type=";
-  private static final String VALIDATION_PATH = "/password/validate";
-  private static final String HEADER_X_OKAPI_TENANT = "x-okapi-tenant";
-  private static final String HEADER_CONTENT_TYPE = "content-type";
-  private static final String HEADER_ACCEPT = "Accept";
-  private static final String ACCEPT_VALUES = "application/json, text/plain";
-  private static final String APPLICATION_JSON = "application/json";
+
+  public static final String ERR_MESSAGE_ID = "errMessageId";
+
+  private static final JsonObject REGEXP_RULE_ONE_LETTER_ONE_NUMBER = new JsonObject()
+    .put("name", "Regexp rule")
+    .put("type", "RegExp")
+    .put("validationType", "Strong")
+    .put("moduleName", "mod-password-validator")
+    .put("expression", "^(?=.*[A-Za-z])(?=.*\\d).+$")
+    .put("description", "At least one letter and one number")
+    .put(ERR_MESSAGE_ID, "password.validation.error.one-letter-one-number");
+
+  private static final JsonObject REGEXP_RULE_MIN_LENGTH_8 = new JsonObject()
+    .put("name", "Regexp rule")
+    .put("type", "RegExp")
+    .put("validationType", "Strong")
+    .put("moduleName", "mod-password-validator")
+    .put("expression", "^.{8,}$")
+    .put("description", "Minimum eight characters")
+    .put(ERR_MESSAGE_ID, "password.validation.error.min-8");
+
+
   private static final String HOST = "http://localhost:";
   private static final String HTTP_PORT = "http.port";
+  private static final String ORDER_NO = "orderNo";
+  private static final String STATE = "state";
+
+  private static final String OKAPI_URL_HEADER = "x-okapi-url";
   private static final String TENANT = "diku";
-  private static final String VALIDATOR_RULE_TYPE_STRONG = "strong";
-  private Vertx vertx;
-  private int port;
+  private static final String VALIDATION_RULES_TABLE_NAME = "validation_rules";
+
+  private static Vertx vertx;
+  private static int port;
+  private static int userMockPort;
+
 
-  @Before
-  public void setUp(final TestContext context) {
+  @Rule
+  public Timeout rule = Timeout.seconds(180);  // 3 minutes for loading embedded postgres
+
+  @BeforeClass
+  public static void setUpClass(final TestContext context) throws IOException {
+    Async async = context.async();
     vertx = Vertx.vertx();
     port = NetworkUtils.nextFreePort();
+    userMockPort = NetworkUtils.nextFreePort();
+
+    PostgresClient.setIsEmbedded(true);
```

Comment:
Consider adding the option to use an existing postgres instance because starting embedded postgres is slow.
See https://github.com/folio-org/mod-inventory-storage/blob/0fda3ab/src/test/java/org/folio/rest/api/StorageTestSuite.java#L70-L89 for an example.